packages/flutter/lib/fix_data/fix_cupertino.yaml

```diff
@@ -295,4 +295,50 @@ transforms:
     - kind: 'rename'
       newName: 'buildOverscrollIndicator'
 
+  # Changes made in https://github.com/flutter/flutter/pull/151367
+  - title: "Rename to 'activeTrackColor'"
+    date: 2024-07-15
+    element:
+      uris: [ 'cupertino.dart' ]
+      field: 'activeColor'
```

Comment:
Test fixes test is failing. This should probably be `getter: 'activeColor'` now.

Reply:
I have changed it to `getter` but it seems to be still failing. Can you please tell me, what I'm doing wrong here ?